openmc/_xml.py

```diff
@@ -0,0 +1,51 @@
+def clean_indentation(element, level=0, spaces_per_level=2):
+    """
+    copy and paste from http://effbot.org/zone/elementent-lib.htm#prettyprint
```

Comment:
The link is now obsolete. Is there a Github or other more permanent link to reference?